packages/rs-dpp/src/data_contract/created_data_contract/v0/mod.rs

```diff
@@ -1,21 +1,24 @@
 use crate::data_contract::data_contract::DataContractV0;
 use crate::data_contract::DataContract;
 
-use crate::data_contract::conversion::platform_value_conversion::v0::DataContractValueConversionMethodsV0;
+#[cfg(feature = "data-contract-value-conversions")]
+use crate::data_contract::conversion::value::v0::DataContractValueConversionMethodsV0;
 use crate::data_contract::created_data_contract::fields::property_names::{DATA_CONTRACT, ENTROPY};
 use crate::version::PlatformVersion;
 use crate::ProtocolError;
 use platform_value::btreemap_extensions::BTreeValueRemoveFromMapHelper;
 use platform_value::{Bytes32, Error, Value};
 
+// TODO: Rename to extended and move metadata here
```

Comment:
Decide on what we need ExtendedDataContract with metadata or CreatedDataContract or both.